lib/ansible/modules/cloud/vmware/vmware_dvs_portgroup.py

```diff
@@ -347,6 +369,55 @@ def check_dvspg_state(self):
         if self.dvs_portgroup is None:
             return 'absent'
         else:
```

Comment:
Since the `if` block below end up with a `return`, the `else` statement is not really useful here. You can just remove it and drop one level of indentation in the rest of the block.

Reply:
Good point, I've remove the `else`.